mcp/src/main/java/io/modelcontextprotocol/server/McpTransportContext.java

```diff
@@ -47,4 +47,13 @@ public interface McpTransportContext {
 	 */
 	McpTransportContext copy();
 
+	/**
+	 * Sends a notification from the server to the client.
+	 * @param method notification method name
+	 * @param params any parameters or {@code null}
+	 */
+	default void sendNotification(String method, Object params) {
```

Comment:
An alternative to this might be a separate `McpNotifier` instance that would be accessed by calling `McpTransportContext.get()` with a lib-defined key. wdyt?